docs/_posts/examples/3400-01-11-mapbox-gl-directions.html

```diff
@@ -0,0 +1,29 @@
+---
+layout: example
+category: example
+title: Driving directions
+description: Use the mapbox-gl-directions plugin to show results from the Mapbox Directions API
+---
+<script src='https://api.tiles.mapbox.com/mapbox-gl-js/plugins/mapbox-gl-directions/v0.0.0/mapbox-gl-directions.js'></script>
+<link rel='stylesheet' href='https://api.tiles.mapbox.com/mapbox-gl-js/plugins/mapbox-gl-directions/v0.0.0/mapbox-gl-directions.css' type='text/css' />
+<div id='map'></div>
+
+<script>
+var map = new mapboxgl.Map({
+  container: 'map',
+  style: 'mapbox://styles/mapbox/streets-v8',
+  center: [-79.4512, 43.6568],
+  zoom: 13
+});
+
+var directions = mapboxgl.Directions(document.getElementById('map'), {
```

Comment:
Is there a reason this uses `document.getElementById('map')` rather than a `container: 'map'` option like `Map`? Can this argument be omitted entirely, and just use the container that the map is using when `map.addControl(directions)` is called?